src/tabs/userManagement/BlacklistWhitelistPromptDialog.tsx

```diff
@@ -67,10 +72,28 @@ export class BlacklistWhitelistPromptDialog extends RX.Component<BlacklistWhitel
     );
   }
 
+  // TODO possibly make this code cleaner
```

Comment:
It's not bad code, however I think it would be preferable to update it from the server instead of performing the same modification locally so that it also covers the case of multiple admins concurrently working on the list from two instances of the web interface.

Later today I'll see if I can manage to get it to automatically update from the server ad we were discussing on Slack